tests/test_plotting.py

```diff
@@ -230,40 +222,27 @@ def test_3d_B(DSHAPE_current):
 
 
 @pytest.mark.unit
-@pytest.mark.solve
-def test_3d_J(DSHAPE_current):
```

Comment:
basically redundant wit the other tests for 3d plotting, we already cover all the coordinate options